packages/fxa-settings/src/pages/CookiesDisabled/en.ftl

```diff
@@ -3,6 +3,7 @@
 
 cookies-disabled-header = Local storage and cookies are required
 cookies-disabled-enable-prompt = Please enable cookies and local storage in your browser to access { -product-firefox-accounts }. Doing so will enable functionality such as remembering you between sessions.
+cookies-disabled-enable-prompt-2 = Please enable cookies and local storage in your browser to access { -product-mozilla-accounts }. Doing so will enable functionality such as remembering you between sessions.
```

Comment:
Should this be using singular or plural? (Since plural is supposed to used when referring to something affecting all Mozilla accounts, not just the individual's account.)

Reply:
Based on feedback, the first sentence will be updated to, "Please enable cookies and local storage in your browser to access your Mozilla account."